api/android/api/src/main/jni/Android.mk

```diff
@@ -89,6 +92,11 @@ NNS_SUBPLUGINS += snpe-subplugin
 include $(LOCAL_PATH)/Android-snpe.mk
 endif
 
+ifeq ($(ENABLE_DECODER_FLATBUF),true)
+include $(LOCAL_PATH)/Android-dec-flatbuf.mk
+NNS_SUBPLUGINS += flatbuffers-subplugin
```

Comment:
should add init function for flatbuf plugin. please see below:
https://github.com/nnstreamer/nnstreamer/blob/386b56722dc7bf39885dc6e8cb644b2455e9d8bf/api/android/api/src/main/jni/nnstreamer-native-api.c#L35

Reply:
@jaeyun-jung Updated. Thanks.